corehq/apps/integration/payments/filters.py

```diff
@@ -0,0 +1,62 @@
+from django.utils.translation import gettext as _
+
+from corehq.apps.es import UserES
+from corehq.apps.reports.filters.base import BaseSingleOptionFilter, BaseMultipleOptionFilter
+from corehq.apps.integration.payments.services import get_payment_batch_numbers_for_domain
+
+
+class PaymentVerificationStatusFilter(BaseSingleOptionFilter):
+    slug = 'payment_verification_status'
+    label = _("Verification Status")
+    default_text = _('Show all')
+
+    verified = 'verified'
+    unverified = 'unverified'
+    options = [
+        (verified, _("Verified")),
+        (unverified, _("Unverified")),
+    ]
+
+
+class BatchNumberFilter(BaseSingleOptionFilter):
+    slug = "batch_number"
+    label = _("Batch number")
+    default_text = _("Show all")
+
+    @property
+    def options(self):
+        batch_numbers = get_payment_batch_numbers_for_domain(self.domain)
+        return [
+            (batch_number, batch_number) for batch_number in batch_numbers
+        ]
+
+
+class PaymentVerifiedByFilter(BaseMultipleOptionFilter):
+    slug = 'verified_by'
+    label = _('Verified by')
+    default_text = _('Show all')
+
+    @property
+    def options(self):
+        query = UserES().domain(self.domain).web_users()
```

Comment:
Its surprising we don't have a web user filter already in HQ.
I'd suggest putting this in `corehq/apps/reports/filters/users.py` where other user filters are. This could be useful for others to use in future.

Reply:
I was too. The closest we have is "Case owners" as far as I can tell, but those include mobile workers.